tests/stt/permissions_test.js

```diff
@@ -116,7 +116,7 @@ Scenario(
     await settingsAPI.apiDisableSTT();
     await I.Authorize(users.admin.username, users.admin.password);
     I.amOnPage(databaseChecksPage.url);
-    I.waitForVisible(databaseChecksPage.fields.dbCheckPanelSelector, 30);
+    I.waitForVisible(databaseChecksPage.fields.dbCheckPanelSelector, 3);
```

Comment:
IMHO that 30 seconds is not a hard wait, it is like a maximum limit, any specific reason for changing this?

Reply:
Yes, tests are run in [parallel](https://github.com/Percona-Lab/jenkins-pipelines/blob/cd0b958b9e505aaa204c8f0a4bf4b03c89d8fe3f/pmm/pmm2-ui-tests.groovy#L282) it is possible to have race conditions, in this case it is better to fail fast and retry.
Of course in some cases it has to be with longer timeout. My point is to use short timeout by default and apply long timeout only when it actually needed.

Reply:
that timeout fails on some API requests that we make in Before tests, so a long timeout will be needed in API requests.

Reply:
got it, I talked to @yurkovychv, instead of changing 30->3 I will introduce constant instead